bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/Guardian/GuardianStorageExtensions.cs

```diff
@@ -17,12 +18,16 @@ public static class GuardianStorageExtensions
         GuardianPrivateRecord privateGuardianRecord,
         string keyCeremonyId,
         int numberOfGuardians,
-        int quorum)
+        int quorum,
+        Dictionary<string, ElectionPublicKey>? publicKeys = null,
+        Dictionary<string, ElectionPartialKeyBackup>? otherBackups = null)
```

Comment:
Needed to add in the publickeys and the other backups duringt he initialization.